spec/default_facts.yml

```diff
@@ -4,3 +4,4 @@ ipaddress: "172.16.254.254"
 is_pe: false
 macaddress: "AA:AA:AA:AA:AA:AA"
 selinux_config_mode: "disabled"
+service_provider: 'dummy'
```

Comment:
this gets overwritten by our modulesync, please move it to default_module_facts.yml